quickstarts/java/rmi/config.yml

```diff
@@ -0,0 +1,51 @@
+id: cfe7983c-fb97-11ed-be56-0242ac120002
```

Comment:
No need to provide an id for the quickstart here, this will be generated and added to the config on first time publishing

Reply:
Removed ID.